App/Redux/IpfsNodeRedux.js

```diff
@@ -28,7 +28,7 @@ export default Creators
 
 export const INITIAL_STATE = Immutable({
   locked: false,
-  appState: 'unknown',
+  appState: null,
```

Comment:
this causes an error on android. i think the native module is expecting a string. should be able to catch it there i think